.github/workflows/ci-macos-combined.yml

```diff
@@ -7,7 +7,7 @@ on:
       - '*'
   pull_request:
     branches:
-      - master
+      - '*'
```

Comment:
You can use `$default-branch` instead if you want support for `main`/`master`

Reply:
I want to support any branch in case of any release branch or collaborative branch with many people doing PRs to said branch

Reply:
Due to the quickcheck change conflict going to leave this change out.